shared/fs/header/header.native.js

```diff
@@ -23,6 +24,20 @@ const Header = ({title, path, onBack}: Props) => (
       <Box style={stylesAddNewBox}>
         <AddNew path={path} style={stylesAddNew} />
       </Box>
+      {onChat && (
+        <Box style={stylesAddNewBox}>
+          <Icon
+            type="iconfont-chat"
+            style={{
+              marginLeft: globalMargins.tiny,
```

Comment:
you don't have to make this change now but we do want to move to using styleSheetCreate everywhere. I'd advocate doing that kind of cleanup when revisiting components. again you don't need to do this in this pr but in general likely a good practice